cmake/Modules/FindXCCL.cmake

```diff
@@ -0,0 +1,68 @@
+# This will define the following variables:
+# XCCL_FOUND               : True if the system has the XCCL library.
+# XCCL_INCLUDE_DIR         : Include directories needed to use XCCL.
+# XCCL_LIBRARY_DIR         ：The path to the XCCL library.
+# XCCL_LIBRARY             : XCCL library fullname.
+
+include(FindPackageHandleStandardArgs)
+
+set(XCCL_ROOT "")
+if(DEFINED ENV{CCL_ROOT})
+  set(XCCL_ROOT $ENV{CCL_ROOT})
+endif()
```

Comment:
What is the relationship between `XCCL_ROOT` and `CCL_ROOT`? 
It looks like `CCL_ROOT` is the user-facing env?

Reply:
Yes, `CCL_ROOT` is a user-facing environment variable that is automatically set by Intel oneCCL.